services/kommander/0.3.2/defaults/cm.yaml

```diff
@@ -86,6 +86,9 @@ data:
           traefik.ingress.kubernetes.io/router.middlewares: "${workspaceNamespace}-stripprefixes@kubernetescrd,${workspaceNamespace}-forwardauth@kubernetescrd"
         path: /dkp/kommander/dashboard
         graphqlPath: /dkp/kommander/dashboard/graphql
+    capimate:
+      image:
+        tag: v0.0.0-dev.0
```

Comment:
@clintonmedbery This version will only work for `main`. Need to take another look at the script and rework it to use the other dev builds that are tied to each release e.g. for `release-2.3` this would be `v2.3.2-dev` now, until `v2.3.2` is released, then it should be bumped to `v2.3.3-dev`. And `main` branch should be pointing to next upcoming release's dev build aka `v2.5.0-dev`

Reply:
I added this commit https://github.com/mesosphere/kommander-applications/pull/905/commits/599ef73776a86fba83883d55ae61948524d27669 which manually updated all the `v0.0.0-dev.0` to `v2.3.2-dev` but the post-release script needs to be updated to bump these versions